src/platform/nrfconnect/FactoryDataProvider.h

```diff
@@ -175,6 +183,9 @@ class FactoryDataProvider : public chip::Credentials::DeviceAttestationCredentia
 
     struct FactoryData mFactoryData;
     FlashFactoryData mFlashFactoryData;
+#ifdef CONFIG_CHIP_CRYPTO_PSA
+    psa_key_id_t mDACPrivKeyId = static_cast<psa_key_id_t>(static_cast<uint32_t>(chip::Crypto::KeyIdBase::Maximum) + 1);
```

Comment:
I think you should rather reserve a value in `KeyIdBase` than assuming `max + 1` is not reserved and can be used for DAC.

Reply:
Ok, I was considering how to do it, and reserving it in KeyIdBase seems good.